Application Loader fails to verify an app with dynamic framework

When trying to upload an app which uses a custom dynamic framework to iTunes Connect using Application Loader I'm getting this error:

ERROR ITMS-90035: "Invalid Signature. Invalid resource directory (directory or signature have been modified). Make sure you have signed your application with a distribution certificate, not an ad hoc certificate or a development certificate. Verify that the code signing settings in Xcode are correct at the target level (which override any values at the project level). Additionally, make sure the bundle you are uploading was built using a Release target in Xcode, not a Simulator target. If you are certain your code signing settings are correct, choose "Clean All" in Xcode, delete the "build" directory in the Finder, and rebuild your release target. For more information, please consult https://developer.apple.com/library/ios/documentation/Security/Conceptual/CodeSigningGuide/Introduction/Introduction.html"

I generated the .ipa using IntelliJ > Build > Create IPA, with a distribution certificate and provisioning profile.

When embedding the same framework in a native xcode app and creating an ipa from that app in xcode (using Archive and then Export), all is working well in Application Loader and the ipa is uploaded successfully.

I've attached a sample project here:
https://drive.google.com/file/d/0BwxZsZfwfINWMHNBTTJRWGFjeUk/view?usp=sharing
The sample contains the robovm project (robovm-ios-appstore), and the xcode project (robovm-ios-appstore/wave) which contains two targets - the framework 'myframework' and the app that uses the framework 'wave'. 
Also there's an 'ipas' folder there which contains both generated ipas, one from xcode and one from robovm.

This is the full robovm console of building the sample ipa with robovm:

[INFO] Creating package in /Users/admin/Desktop/robovm-ios-appstore/appstore ...
[DEBUG] Loading default RoboVM config properties file: /Users/admin/Desktop/robovm-ios-appstore/robovm.properties
[DEBUG] Loading default RoboVM config file: /Users/admin/Desktop/robovm-ios-appstore/robovm.xml
[INFO] classpath entry: /Users/admin/Desktop/robovm-ios-appstore/build/resources/main
[INFO] classpath entry: /Users/admin/Desktop/robovm-ios-appstore/build/classes/main
[INFO] Using SDK boot classpath
[INFO] Creating thumbv7 slice for IPA
[DEBUG] Compiling classes using 8 threads
[DEBUG] Compiling media.muvix.wave.Main (ios thumbv7 release)
[DEBUG] Compiling media.muvix.wave.MyViewController (ios thumbv7 release)
[DEBUG] Compiling org.robovm.objc.annotation.IBAction (ios thumbv7 release)
[DEBUG] Compiling org.robovm.objc.annotation.IBOutlet (ios thumbv7 release)
[DEBUG] Compiling org.robovm.objc.annotation.IBAction$Impl (ios thumbv7 release)
[DEBUG] Compiling org.robovm.objc.annotation.IBOutlet$Impl (ios thumbv7 release)
[DEBUG] Compiled 6 classes in 3.40 seconds
[INFO] Linking 2355 classes (ios thumbv7 release)
[DEBUG] 29220 methods out of 29220 included in the executable
[DEBUG] Building executable /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/thumbv7/Muvix
[DEBUG] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -o /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/thumbv7/Muvix -arch armv7 -Wl,-filelist,/Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/thumbv7/objects0 -L /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/thumbv7 -ObjC -exported_symbols_list /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/thumbv7/exported_symbols -Wl,-no_implicit_dylibs -Wl,-dead_strip -fPIC -miphoneos-version-min=8.0 -stdlib=libstdc++ -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS8.4.sdk -lrobovm-bc -force_load /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/thumbv7/librobovm-rt.a -lrobovm-core -lgc -lpthread -ldl -lm -liconv -lsqlite3 -framework Foundation -framework MobileCoreServices -framework UIKit -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks
[DEBUG] Linked 2355 classes in 2.33 seconds
[INFO] Creating arm64 slice for IPA
[DEBUG] Compiling classes using 8 threads
[DEBUG] Compiling media.muvix.wave.Main (ios arm64 release)
[DEBUG] Compiling media.muvix.wave.MyViewController (ios arm64 release)
[DEBUG] Compiling org.robovm.objc.annotation.IBAction (ios arm64 release)
[DEBUG] Compiling org.robovm.objc.annotation.IBOutlet (ios arm64 release)
[DEBUG] Compiling org.robovm.objc.annotation.IBAction$Impl (ios arm64 release)
[DEBUG] Compiling org.robovm.objc.annotation.IBOutlet$Impl (ios arm64 release)
[DEBUG] Compiled 6 classes in 2.87 seconds
[INFO] Linking 2355 classes (ios arm64 release)
[DEBUG] 29220 methods out of 29220 included in the executable
[DEBUG] Building executable /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/arm64/Muvix
[DEBUG] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -o /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/arm64/Muvix -arch arm64 -Wl,-filelist,/Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/arm64/objects0 -L /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/arm64 -ObjC -exported_symbols_list /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/arm64/exported_symbols -Wl,-no_implicit_dylibs -Wl,-dead_strip -fPIC -miphoneos-version-min=8.0 -stdlib=libstdc++ -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS8.4.sdk -lrobovm-bc -force_load /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/arm64/librobovm-rt.a -lrobovm-core -lgc -lpthread -ldl -lm -liconv -lsqlite3 -framework Foundation -framework MobileCoreServices -framework UIKit -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks
[DEBUG] Linked 2355 classes in 2.28 seconds
[DEBUG] Creating IPA in /Users/admin/Desktop/robovm-ios-appstore/appstore
[DEBUG] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/thumbv7/Muvix /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/arm64/Muvix -create -output /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/Muvix
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/boot/robovm-rt.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/classes0.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/robovm-cacerts-full.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/robovm-cocoatouch.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/robovm-objc.jar
[DEBUG] /Applications/Xcode.app/Contents/Developer/usr/bin/ibtool --target-device iphone --target-device ipad --minimum-deployment-target 8.0 --output-partial-info-plist /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/partial-plists/Main.storyboard_8099495678022982194.plist --auto-activate-custom-fonts --output-format human-readable-text --compilation-directory /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/Base.lproj /Users/admin/Desktop/robovm-ios-appstore/resources/Base.lproj/Main.storyboard
[DEBUG] /Applications/Xcode.app/Contents/Developer/usr/bin/actool --output-format human-readable-text --app-icon AppIcon --launch-image LaunchImage --output-partial-info-plist /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/partial-plists/Images.xcassets_8899567715731925855.plist --platform iphoneos --minimum-deployment-target 8.0 --target-device iphone --target-device ipad --compress-pngs --compile /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app /Users/admin/Desktop/robovm-ios-appstore/resources/Images.xcassets
[DEBUG] /* com.apple.actool.compilation-results */
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon29x29@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon29x29@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon40x40@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon40x40@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon60x60@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon60x60@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon29x29~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon29x29@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon40x40~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon40x40@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon76x76~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon76x76@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-800-Portrait-736h@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-800-Landscape-736h@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-800-667h@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-568h@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-Portrait~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-Landscape~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-Portrait@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-Landscape@2x~ipad.png
[DEBUG] /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/partial-plists/Images.xcassets_8899567715731925855.plist
[DEBUG] 
[DEBUG] Installing Info.plist to /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app
[DEBUG] xcrun dsymutil -o /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app.dSYM /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/Muvix
[DEBUG] xcrun strip -x /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/Muvix
[DEBUG] Copying AppStore provisioning profile: Muvix Distribution Profile (NT87EN6678.media.muvix.wave)
[DEBUG] Code signing using identity 'iPhone Distribution: ******* (*******)' with fingerprint 6CB85E3B3CC8DAED97C18C6076A33E1FAAF821AA
[DEBUG] codesign -f -s 6CB85E3B3CC8DAED97C18C6076A33E1FAAF821AA --entitlements /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5171562415026159701.tmp/Entitlements.plist /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app
[DEBUG] ln -f -s _CodeSignature/CodeResources /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/CodeResources
[DEBUG] /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/PackageApplication /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app -o /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.ipa
[INFO] Package successfully created in /Users/admin/Desktop/robovm-ios-appstore/appstore
[INFO] IPA creation complete, 0 errors, 0 warnings
[INFO] Creating package in /Users/admin/Desktop/robovm-ios-appstore/appstore ...
[DEBUG] Loading default RoboVM config properties file: /Users/admin/Desktop/robovm-ios-appstore/robovm.properties
[DEBUG] Loading default RoboVM config file: /Users/admin/Desktop/robovm-ios-appstore/robovm.xml
[INFO] classpath entry: /Users/admin/Desktop/robovm-ios-appstore/build/resources/main
[INFO] classpath entry: /Users/admin/Desktop/robovm-ios-appstore/build/classes/main
[INFO] Using SDK boot classpath
[INFO] Creating thumbv7 slice for IPA
[DEBUG] Compiling classes using 8 threads
[DEBUG] Compiling media.muvix.wave.Main (ios thumbv7 release)
[DEBUG] Compiling media.muvix.wave.MyViewController (ios thumbv7 release)
[DEBUG] Compiled 2 classes in 2.83 seconds
[INFO] Linking 2355 classes (ios thumbv7 release)
[DEBUG] 29220 methods out of 29220 included in the executable
[DEBUG] Building executable /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/thumbv7/Muvix
[DEBUG] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -o /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/thumbv7/Muvix -arch armv7 -Wl,-filelist,/Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/thumbv7/objects0 -L /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/thumbv7 -ObjC -exported_symbols_list /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/thumbv7/exported_symbols -Wl,-no_implicit_dylibs -Wl,-dead_strip -F/Users/admin/Desktop/robovm-ios-appstore/frameworks -fPIC -miphoneos-version-min=8.0 -stdlib=libstdc++ -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS8.4.sdk -lrobovm-bc -force_load /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/thumbv7/librobovm-rt.a -lrobovm-core -lgc -lpthread -ldl -lm -liconv -lsqlite3 -framework Foundation -framework myframework -framework MobileCoreServices -framework UIKit -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks
[DEBUG] Linked 2355 classes in 2.18 seconds
[INFO] Creating arm64 slice for IPA
[DEBUG] Compiling classes using 8 threads
[DEBUG] Compiling media.muvix.wave.Main (ios arm64 release)
[DEBUG] Compiling media.muvix.wave.MyViewController (ios arm64 release)
[DEBUG] Compiled 2 classes in 3.03 seconds
[INFO] Linking 2355 classes (ios arm64 release)
[DEBUG] 29220 methods out of 29220 included in the executable
[DEBUG] Building executable /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/arm64/Muvix
[DEBUG] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -o /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/arm64/Muvix -arch arm64 -Wl,-filelist,/Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/arm64/objects0 -L /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/arm64 -ObjC -exported_symbols_list /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/arm64/exported_symbols -Wl,-no_implicit_dylibs -Wl,-dead_strip -F/Users/admin/Desktop/robovm-ios-appstore/frameworks -fPIC -miphoneos-version-min=8.0 -stdlib=libstdc++ -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS8.4.sdk -lrobovm-bc -force_load /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/arm64/librobovm-rt.a -lrobovm-core -lgc -lpthread -ldl -lm -liconv -lsqlite3 -framework Foundation -framework myframework -framework MobileCoreServices -framework UIKit -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks
[DEBUG] Linked 2355 classes in 2.09 seconds
[DEBUG] Creating IPA in /Users/admin/Desktop/robovm-ios-appstore/appstore
[DEBUG] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/thumbv7/Muvix /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/arm64/Muvix -create -output /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/Muvix
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/boot/robovm-rt.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/classes0.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/robovm-cacerts-full.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/robovm-cocoatouch.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/robovm-objc.jar
[DEBUG] /Applications/Xcode.app/Contents/Developer/usr/bin/ibtool --target-device iphone --target-device ipad --minimum-deployment-target 8.0 --output-partial-info-plist /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/partial-plists/Main.storyboard_8836635663693321114.plist --auto-activate-custom-fonts --output-format human-readable-text --compilation-directory /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/Base.lproj /Users/admin/Desktop/robovm-ios-appstore/resources/Base.lproj/Main.storyboard
[DEBUG] /Applications/Xcode.app/Contents/Developer/usr/bin/actool --output-format human-readable-text --app-icon AppIcon --launch-image LaunchImage --output-partial-info-plist /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/partial-plists/Images.xcassets_5926038446910683187.plist --platform iphoneos --minimum-deployment-target 8.0 --target-device iphone --target-device ipad --compress-pngs --compile /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app /Users/admin/Desktop/robovm-ios-appstore/resources/Images.xcassets
[DEBUG] /* com.apple.actool.compilation-results */
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon29x29@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon29x29@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon40x40@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon40x40@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon60x60@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon60x60@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon29x29~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon29x29@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon40x40~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon40x40@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon76x76~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon76x76@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-800-Portrait-736h@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-800-Landscape-736h@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-800-667h@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-568h@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-Portrait~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-Landscape~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-Portrait@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-Landscape@2x~ipad.png
[DEBUG] /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/partial-plists/Images.xcassets_5926038446910683187.plist
[DEBUG] 
[DEBUG] Copying framework myframework from /Users/admin/Desktop/robovm-ios-appstore/frameworks/myframework.framework to /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app
[DEBUG] Copying resource /Users/admin/Desktop/robovm-ios-appstore/frameworks/myframework.framework/Info.plist to /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/Frameworks/myframework.framework
[DEBUG] Copying resource /Users/admin/Desktop/robovm-ios-appstore/frameworks/myframework.framework/myframework to /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/Frameworks/myframework.framework
[DEBUG] Installing Info.plist to /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app
[DEBUG] xcrun dsymutil -o /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app.dSYM /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/Muvix
[DEBUG] xcrun strip -x /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/Muvix
[DEBUG] Copying AppStore provisioning profile: Muvix Distribution Profile (NT87EN6678.media.muvix.wave)
[DEBUG] Code signing using identity 'iPhone Distribution: ****** (********)' with fingerprint 6CB85E3B3CC8DAED97C18C6076A33E1FAAF821AA
[DEBUG] codesign -f -s 6CB85E3B3CC8DAED97C18C6076A33E1FAAF821AA --entitlements /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5154406277133211596.tmp/Entitlements.plist /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app
[DEBUG] ln -f -s _CodeSignature/CodeResources /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/CodeResources
[DEBUG] /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/PackageApplication /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app -o /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.ipa
[INFO] Package successfully created in /Users/admin/Desktop/robovm-ios-appstore/appstore
[INFO] IPA creation complete, 0 errors, 0 warnings
[INFO] Creating package in /Users/admin/Desktop/robovm-ios-appstore/appstore ...
[DEBUG] Loading default RoboVM config properties file: /Users/admin/Desktop/robovm-ios-appstore/robovm.properties
[DEBUG] Loading default RoboVM config file: /Users/admin/Desktop/robovm-ios-appstore/robovm.xml
[INFO] classpath entry: /Users/admin/Desktop/robovm-ios-appstore/build/resources/main
[INFO] classpath entry: /Users/admin/Desktop/robovm-ios-appstore/build/classes/main
[INFO] Using SDK boot classpath
[INFO] Creating thumbv7 slice for IPA
[DEBUG] Compiling classes using 8 threads
[DEBUG] Compiling media.muvix.wave.Main (ios thumbv7 release)
[DEBUG] Compiling media.muvix.wave.MyViewController (ios thumbv7 release)
[DEBUG] Compiled 2 classes in 1.80 seconds
[INFO] Linking 2355 classes (ios thumbv7 release)
[DEBUG] 29220 methods out of 29220 included in the executable
[DEBUG] Building executable /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/thumbv7/Muvix
[DEBUG] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -o /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/thumbv7/Muvix -arch armv7 -Wl,-filelist,/Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/thumbv7/objects0 -L /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/thumbv7 -ObjC -exported_symbols_list /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/thumbv7/exported_symbols -Wl,-no_implicit_dylibs -Wl,-dead_strip -F/Users/admin/Desktop/robovm-ios-appstore/frameworks -fPIC -miphoneos-version-min=8.0 -stdlib=libstdc++ -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS8.4.sdk -lrobovm-bc -force_load /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/thumbv7/librobovm-rt.a -lrobovm-core -lgc -lpthread -ldl -lm -liconv -lsqlite3 -framework Foundation -framework myframework -framework MobileCoreServices -framework UIKit -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks
[DEBUG] Linked 2355 classes in 1.92 seconds
[INFO] Creating arm64 slice for IPA
[DEBUG] Compiling classes using 8 threads
[DEBUG] Compiling media.muvix.wave.Main (ios arm64 release)
[DEBUG] Compiling media.muvix.wave.MyViewController (ios arm64 release)
[DEBUG] Compiled 2 classes in 1.69 seconds
[INFO] Linking 2355 classes (ios arm64 release)
[DEBUG] 29220 methods out of 29220 included in the executable
[DEBUG] Building executable /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/arm64/Muvix
[DEBUG] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -o /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/arm64/Muvix -arch arm64 -Wl,-filelist,/Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/arm64/objects0 -L /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/arm64 -ObjC -exported_symbols_list /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/arm64/exported_symbols -Wl,-no_implicit_dylibs -Wl,-dead_strip -F/Users/admin/Desktop/robovm-ios-appstore/frameworks -fPIC -miphoneos-version-min=8.0 -stdlib=libstdc++ -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS8.4.sdk -lrobovm-bc -force_load /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/arm64/librobovm-rt.a -lrobovm-core -lgc -lpthread -ldl -lm -liconv -lsqlite3 -framework Foundation -framework myframework -framework MobileCoreServices -framework UIKit -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks
[DEBUG] Linked 2355 classes in 1.83 seconds
[DEBUG] Creating IPA in /Users/admin/Desktop/robovm-ios-appstore/appstore
[DEBUG] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/thumbv7/Muvix /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/arm64/Muvix -create -output /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/Muvix
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/boot/robovm-rt.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/classes0.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/robovm-cacerts-full.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/robovm-cocoatouch.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/lib/robovm-objc.jar
[DEBUG] /Applications/Xcode.app/Contents/Developer/usr/bin/ibtool --target-device iphone --target-device ipad --minimum-deployment-target 8.0 --output-partial-info-plist /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/partial-plists/Main.storyboard_8806011351880844441.plist --auto-activate-custom-fonts --output-format human-readable-text --compilation-directory /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/Base.lproj /Users/admin/Desktop/robovm-ios-appstore/resources/Base.lproj/Main.storyboard
[DEBUG] /Applications/Xcode.app/Contents/Developer/usr/bin/actool --output-format human-readable-text --app-icon AppIcon --launch-image LaunchImage --output-partial-info-plist /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/partial-plists/Images.xcassets_6451757868205236637.plist --platform iphoneos --minimum-deployment-target 8.0 --target-device iphone --target-device ipad --compress-pngs --compile /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app /Users/admin/Desktop/robovm-ios-appstore/resources/Images.xcassets
[DEBUG] /* com.apple.actool.compilation-results */
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon29x29@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon29x29@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon40x40@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon40x40@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon60x60@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon60x60@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon29x29~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon29x29@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon40x40~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon40x40@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon76x76~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/AppIcon76x76@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-800-Portrait-736h@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-800-Landscape-736h@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-800-667h@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-568h@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-Portrait~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-Landscape~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-Portrait@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/LaunchImage-700-Landscape@2x~ipad.png
[DEBUG] /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/partial-plists/Images.xcassets_6451757868205236637.plist
[DEBUG] 
[DEBUG] Copying framework myframework from /Users/admin/Desktop/robovm-ios-appstore/frameworks/myframework.framework to /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app
[DEBUG] Copying resource /Users/admin/Desktop/robovm-ios-appstore/frameworks/myframework.framework/Info.plist to /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/Frameworks/myframework.framework
[DEBUG] Copying resource /Users/admin/Desktop/robovm-ios-appstore/frameworks/myframework.framework/myframework to /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/Frameworks/myframework.framework
[DEBUG] Installing Info.plist to /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app
[DEBUG] xcrun dsymutil -o /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app.dSYM /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/Muvix
[DEBUG] xcrun strip -x /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/Muvix
[DEBUG] Copying AppStore provisioning profile: Muvix Distribution Profile (NT87EN6678.media.muvix.wave)
[DEBUG] Code signing using identity 'iPhone Distribution: ****** (*****)' with fingerprint 6CB85E3B3CC8DAED97C18C6076A33E1FAAF821AA
[DEBUG] codesign -f -s 6CB85E3B3CC8DAED97C18C6076A33E1FAAF821AA --entitlements /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm5262264610907390910.tmp/Entitlements.plist /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app
[DEBUG] ln -f -s _CodeSignature/CodeResources /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app/CodeResources
[DEBUG] /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/PackageApplication /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.app -o /Users/admin/Desktop/robovm-ios-appstore/appstore/Muvix.ipa
[INFO] Package successfully created in /Users/admin/Desktop/robovm-ios-appstore/appstore
[INFO] IPA creation complete, 0 errors, 0 warnings
[INFO] Creating package in /Users/admin/Desktop/robovm-ios-appstore/ipas ...
[DEBUG] Loading default RoboVM config properties file: /Users/admin/Desktop/robovm-ios-appstore/robovm.properties
[DEBUG] Loading default RoboVM config file: /Users/admin/Desktop/robovm-ios-appstore/robovm.xml
[INFO] classpath entry: /Users/admin/Desktop/robovm-ios-appstore/build/resources/main
[INFO] classpath entry: /Users/admin/Desktop/robovm-ios-appstore/build/classes/main
[INFO] Using SDK boot classpath
[INFO] Creating thumbv7 slice for IPA
[DEBUG] Compiling classes using 8 threads
[DEBUG] Compiling media.muvix.wave.Main (ios thumbv7 release)
[DEBUG] Compiling media.muvix.wave.MyViewController (ios thumbv7 release)
[DEBUG] Compiled 2 classes in 2.88 seconds
[INFO] Linking 2355 classes (ios thumbv7 release)
[DEBUG] 29220 methods out of 29220 included in the executable
[DEBUG] Building executable /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/thumbv7/Muvix
[DEBUG] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -o /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/thumbv7/Muvix -arch armv7 -Wl,-filelist,/Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/thumbv7/objects0 -L /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/thumbv7 -ObjC -exported_symbols_list /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/thumbv7/exported_symbols -Wl,-no_implicit_dylibs -Wl,-dead_strip -F/Users/admin/Desktop/robovm-ios-appstore/frameworks -fPIC -miphoneos-version-min=8.0 -stdlib=libstdc++ -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS8.4.sdk -lrobovm-bc -force_load /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/thumbv7/librobovm-rt.a -lrobovm-core -lgc -lpthread -ldl -lm -liconv -lsqlite3 -framework Foundation -framework myframework -framework MobileCoreServices -framework UIKit -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks
[DEBUG] Linked 2355 classes in 2.20 seconds
[INFO] Creating arm64 slice for IPA
[DEBUG] Compiling classes using 8 threads
[DEBUG] Compiling media.muvix.wave.Main (ios arm64 release)
[DEBUG] Compiling media.muvix.wave.MyViewController (ios arm64 release)
[DEBUG] Compiled 2 classes in 3.00 seconds
[INFO] Linking 2355 classes (ios arm64 release)
[DEBUG] 29220 methods out of 29220 included in the executable
[DEBUG] Building executable /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/arm64/Muvix
[DEBUG] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -o /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/arm64/Muvix -arch arm64 -Wl,-filelist,/Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/arm64/objects0 -L /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/arm64 -ObjC -exported_symbols_list /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/arm64/exported_symbols -Wl,-no_implicit_dylibs -Wl,-dead_strip -F/Users/admin/Desktop/robovm-ios-appstore/frameworks -fPIC -miphoneos-version-min=8.0 -stdlib=libstdc++ -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS8.4.sdk -lrobovm-bc -force_load /Users/admin/.robovm-sdks/robovm-1.5.0/lib/vm/ios/arm64/librobovm-rt.a -lrobovm-core -lgc -lpthread -ldl -lm -liconv -lsqlite3 -framework Foundation -framework myframework -framework MobileCoreServices -framework UIKit -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks
[DEBUG] Linked 2355 classes in 2.14 seconds
[DEBUG] Creating IPA in /Users/admin/Desktop/robovm-ios-appstore/ipas
[DEBUG] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/thumbv7/Muvix /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/arm64/Muvix -create -output /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/Muvix
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/lib/boot/robovm-rt.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/lib/classes0.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/lib/robovm-cacerts-full.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/lib/robovm-cocoatouch.jar
[DEBUG] Creating stripped archive file /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/lib/robovm-objc.jar
[DEBUG] /Applications/Xcode.app/Contents/Developer/usr/bin/ibtool --target-device iphone --target-device ipad --minimum-deployment-target 8.0 --output-partial-info-plist /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/partial-plists/Main.storyboard_165738241923319142.plist --auto-activate-custom-fonts --output-format human-readable-text --compilation-directory /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/Base.lproj /Users/admin/Desktop/robovm-ios-appstore/resources/Base.lproj/Main.storyboard
[DEBUG] /Applications/Xcode.app/Contents/Developer/usr/bin/actool --output-format human-readable-text --app-icon AppIcon --launch-image LaunchImage --output-partial-info-plist /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/partial-plists/Images.xcassets_373969448938812097.plist --platform iphoneos --minimum-deployment-target 8.0 --target-device iphone --target-device ipad --compress-pngs --compile /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app /Users/admin/Desktop/robovm-ios-appstore/resources/Images.xcassets
[DEBUG] /* com.apple.actool.compilation-results */
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/AppIcon29x29@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/AppIcon29x29@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/AppIcon40x40@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/AppIcon40x40@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/AppIcon60x60@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/AppIcon60x60@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/AppIcon29x29~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/AppIcon29x29@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/AppIcon40x40~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/AppIcon40x40@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/AppIcon76x76~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/AppIcon76x76@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/LaunchImage-800-Portrait-736h@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/LaunchImage-800-Landscape-736h@3x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/LaunchImage-800-667h@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/LaunchImage-700@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/LaunchImage-700-568h@2x.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/LaunchImage-700-Portrait~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/LaunchImage-700-Landscape~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/LaunchImage-700-Portrait@2x~ipad.png
[DEBUG] /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/LaunchImage-700-Landscape@2x~ipad.png
[DEBUG] /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/partial-plists/Images.xcassets_373969448938812097.plist
[DEBUG] 
[DEBUG] Copying framework myframework from /Users/admin/Desktop/robovm-ios-appstore/frameworks/myframework.framework to /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app
[DEBUG] Copying resource /Users/admin/Desktop/robovm-ios-appstore/frameworks/myframework.framework/Info.plist to /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/Frameworks/myframework.framework
[DEBUG] Copying resource /Users/admin/Desktop/robovm-ios-appstore/frameworks/myframework.framework/myframework to /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/Frameworks/myframework.framework
[DEBUG] Installing Info.plist to /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app
[DEBUG] xcrun dsymutil -o /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app.dSYM /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/Muvix
[DEBUG] xcrun strip -x /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/Muvix
[DEBUG] Copying AppStore provisioning profile: Muvix Distribution Profile (NT87EN6678.media.muvix.wave)
[DEBUG] Code signing using identity 'iPhone Distribution: ******* (*******)' with fingerprint 6CB85E3B3CC8DAED97C18C6076A33E1FAAF821AA
[DEBUG] codesign -f -s 6CB85E3B3CC8DAED97C18C6076A33E1FAAF821AA --entitlements /Users/admin/Library/Caches/IntelliJIdea14/tmp/robovm1345671441163966478.tmp/Entitlements.plist /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app
[DEBUG] ln -f -s _CodeSignature/CodeResources /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app/CodeResources
[DEBUG] /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin/PackageApplication /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.app -o /Users/admin/Desktop/robovm-ios-appstore/ipas/Muvix.ipa
[INFO] Package successfully created in /Users/admin/Desktop/robovm-ios-appstore/ipas
[INFO] IPA creation complete, 0 errors, 0 warnings

I'm getting this error when building robovm-idea:

robovm-idea/src/main/java/org/robovm/idea/compilation/RoboVmCompileTask.java:[323,40] error: cannot find symbol
[ERROR] symbol:   variable TYPE
[ERROR] location: class IOSTarget

Where is this symbol?

You can ignore these artifacts as well. They are all commercial and not
deployed to SonaType or available from Github.

That symbol is in the compiler. Did you do a mvn clean install in the
robovm/ project?

On Tue, Jul 14, 2015 at 3:06 PM, alexitelman <notifications@github.com>
wrote:

> I'm getting this error when building robovm-idea:
>
> robovm-idea/src/main/java/org/robovm/idea/compilation/RoboVmCompileTask.java:[323,40]
> error: cannot find symbol
> [ERROR] symbol: variable TYPE
> [ERROR] location: class IOSTarget
>
> Where is this symbol?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/robovm/robovm/issues/1062#issuecomment-121233362>.
>

--- a/compiler/src/main/java/org/robovm/compiler/target/ios/IOSTarget.java
+++ b/compiler/src/main/java/org/robovm/compiler/target/ios/IOSTarget.java
@@ -361,7 +361,7 @@ public class IOSTarget extends AbstractTarget {
                 copyProvisioningProfile(provisioningProfile, installDir);
                 boolean getTaskAllow = provisioningProfile.getType() == Type.Development;
                 signFrameworks(installDir, getTaskAllow);
-                codesign(signIdentity, getOrCreateEntitlementsPList(getTaskAllow), installDir);
+                codesign(signIdentity, getOrCreateEntitlementsPList(getTaskAllow, getBundleId()), installDir);
                 // For some odd reason there needs to be a symbolic link in the
                 // root of
                 // the app bundle named CodeResources pointing at
@@ -391,12 +391,12 @@ public class IOSTarget extends AbstractTarget {
             if (config.isIosSkipSigning()) {
                 config.getLogger().warn("Skiping code signing. The resulting app will "
                         + "be unsigned and will not run on unjailbroken devices");
-                ldid(getOrCreateEntitlementsPList(true), appDir);
+                ldid(getOrCreateEntitlementsPList(true, getBundleId()), appDir);
             } else {
                 copyProvisioningProfile(provisioningProfile, appDir);
                 signFrameworks(appDir, true);
                 // sign the app
-                codesign(signIdentity, getOrCreateEntitlementsPList(true), appDir);
+                codesign(signIdentity, getOrCreateEntitlementsPList(true, getBundleId()), appDir);
             }
         }
     }
@@ -408,17 +408,32 @@ public class IOSTarget extends AbstractTarget {
             // Sign swift rt libs
             for (File swiftLib : frameworksDir.listFiles()) {
                 if (swiftLib.getName().endsWith(".dylib")) {
-                    codesign(signIdentity, getOrCreateEntitlementsPList(getTaskAllow), swiftLib);
+                    codesign(signIdentity, getOrCreateEntitlementsPList(getTaskAllow, getBundleId()), swiftLib);
                 }
             }
 
             // sign embedded frameworks
             for (File framework : frameworksDir.listFiles()) {
                 if (framework.isDirectory() && framework.getName().endsWith(".framework")) {
-                    codesign(signIdentity, getOrCreateEntitlementsPList(getTaskAllow), framework);
+                    codesign(signIdentity, getOrCreateEntitlementsPList(getTaskAllow, getFrameworkBundleId(framework)), framework);
                 }
             }
         }
+    }
+
+    private String getFrameworkBundleId(File frameworkDir) {
+        File frameworkInfoPListFile = new File(frameworkDir, "Info.plist");
+        if (!frameworkInfoPListFile.exists()){
+            throw new Error(String.format("Couldn't find Info.plist in framework %s", frameworkDir));
+        }
+
+        InfoPList frameworkInfoPList = new InfoPList(frameworkInfoPListFile);
+        String bundleIdentifier = frameworkInfoPList.getBundleIdentifier();
+        if (bundleIdentifier == null){
+            throw new Error(String.format("Couldn't find Bundle Identifier in Info.plist in framework %s", frameworkDir));
+        }
+
+        return bundleIdentifier;
     }
 
     private void codesign(SigningIdentity identity, File entitlementsPList, File appDir) throws IOException {
@@ -463,7 +478,7 @@ public class IOSTarget extends AbstractTarget {
         }
     }
 
-    private File getOrCreateEntitlementsPList(boolean getTaskAllow) throws IOException {
+    private File getOrCreateEntitlementsPList(boolean getTaskAllow, String bundleId) throws IOException {
         try {
             File destFile = new File(config.getTmpDir(), "Entitlements.plist");
             NSDictionary dict = null;
@@ -480,7 +495,7 @@ public class IOSTarget extends AbstractTarget {
                         dict.put(key, profileEntitlements.objectForKey(key));
                     }
                 }
-                dict.put("application-identifier", provisioningProfile.getAppIdPrefix() + "." + getBundleId());
+                dict.put("application-identifier", provisioningProfile.getAppIdPrefix() + "." + bundleId);
             }
             dict.put("get-task-allow", getTaskAllow);
             PropertyListParser.saveAsXML(dict, destFile);

